package.json

```diff
@@ -1,47 +1,48 @@
 {
-	"devDependencies": {
-		"babel-core": "^6.26.0",
-		"babel-jest": "^22.1.0",
-		"babel-loader": "^7.1.3",
-		"babel-preset-cozy-app": "^0.8.0",
-		"babel-preset-react": "^6.24.1",
-		"enzyme": "^3.3.0",
-		"enzyme-adapter-react-16": "^1.1.1",
-		"eslint": "^4.16.0",
-		"eslint-config-cozy-app": "^0.8.0",
-		"eslint-loader": "^2.0.0",
-		"eslint-plugin-prettier": "^2.6.0",
-		"jest": "^22.1.4",
-		"jest-fetch-mock": "^1.4.1",
-		"lerna": "^2.8.0",
-		"prettier": "1.10.2",
-		"react": "^16.2.0",
-		"react-dom": "^16.2.0",
-		"redux-mock-store": "^1.5.1",
-		"webpack": "^4.0.1",
-		"webpack-cli": "^2.0.9"
-	},
-	"private": true,
-	"workspaces": [
-		"packages/*"
-	],
-	"scripts": {
-		"lint": "eslint 'packages/*/src/**/*.js'",
-		"test": "jest",
-		"watch": "lerna run watch",
-		"build": "lerna run build"
-	},
-	"prettier": {
-		"semi": false,
-		"singleQuote": true
-	},
-	"jest": {
-		"setupFiles": [
-			"<rootDir>/packages/cozy-client/src/__tests__/setup.js"
-		],
-		"setupTestFrameworkScriptFile": "<rootDir>/packages/cozy-stack-client/src/__tests__/setup.js",
-		"testMatch": [
-			"**/__tests__/**/(*.)(spec|test).js?(x)"
-		]
-	}
+  "name": "cozy-client",
```

Comment:
This name is used in `./packages/cozy-client/package.json` file: https://github.com/cozy/cozy-client/blob/master/packages/cozy-client/package.json#L2